addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1297,10 +1312,54 @@ var SnippetsMenu = Widget.extend({
                     $snippet.addClass('o_snippet_install');
                     var $installBtn = $('<a/>', {
                         class: 'btn btn-primary o_install_btn',
-                        target: '_blank',
-                        href: '/web#id=' + moduleID + '&view_type=form&model=ir.module.module&action=base.open_module_tree',
+                        href: '#',
                         text: _t("Install"),
                     });
+                    $installBtn.on('click', function (ev) {
+                        new Dialog(this, {
+                            title: _.str.sprintf(_t('Install "%s"'), name),
+                            size: 'medium',
+                            $content: $('<p/>', {text: _.str.sprintf(_t('Do you want to install the "%s" App? '), name)}).append(
+                                $('<a/>', {
+                                    target: '_blank',
+                                    href: '/web#id=' + moduleID + '&view_type=form&model=ir.module.module&action=base.open_module_tree',
+                                    text: _t("More info about this app."),
+                                })
+                            ),
+                            buttons: [{
+                                text: _t("Save and Install"),
+                                classes: 'btn-primary',
+                                click: function () {
+                                    var $dialog = this;
```

Comment:
:man_shrugging:

Reply:
`this` is the `Dialog` instance, not a jQuery element. So in worst case, you could name this variable `dialog`... but you should simply use `this`